File: assignments/a1/q1/Java/Skip.java
//Skip:top//
//Skip:import//
import java.util.*;

// <skip> ::= POUND <sentence>
public class Skip /*Skip:class*/ {

    public static final String $className = "Skip";
    public static final String $ruleString =
        "<skip> ::= POUND <sentence>";

    public Sentence sentence;

    public Skip(Sentence sentence) {
//Skip:init//
        this.sentence = sentence;
    }

    public static Skip parse(Scan scn$, Trace trace$) {
        if (trace$ != null)
            trace$ = trace$.nonterm("<skip>", scn$.lno);
        scn$.match(Token.Match.POUND, trace$);
        Sentence sentence = Sentence.parse(scn$, trace$);
        return new Skip(sentence);
    }

//Skip//
}

File: assignments/a1/q1/Python/Skip.py
#Skip:top#
#Skip:import#

# <skip> ::= POUND <sentence>
class Skip(): #Skip:class#

    className = "Skip"
    ruleString = "<skip> ::= POUND <sentence>"
    sentence = None

    def __init__(sentence):
        #Skip:init#
        self.sentence = sentence

#Skip#
